### ISSUE TITLE
Lint doesn't seem to follow default configuration

### ISSUE THREAD

**Issue by ttshivers** [ahmadnassri/action-conventional-commit-lint#24]
Action says it follows `@commitlint/config-conventional` and it looks like it does checking out the source. 

Here is a commit that it says is invalid but actually is:
```
feat!: add ice cream
```

Errors:
```
::error::✖ subject may not be empty
::error::✖ type may not be empty
```

Pulling down your action, installing the deps and even writing down a very paired down version yields the same result:
```js
const { default: lint } = require('@commitlint/lint')
const { parserPreset, rules } = require('@commitlint/config-conventional')

const run = async () => {
  const message = `feat!: add ice cream`;
  const result = await lint(message, rules, parserPreset)
  console.log(result)
};

run();
```

I printed out the parserPreset and rules and they match `@commitlint/config-conventional`  exactly.

Running this also reports no problems:
```sh
echo 'feat!: add ice cream' | npx commitlint --parser-preset conventional-changelog-conventionalcommits --extends '@commitlint/config-conventional'
```

I'm stumped why its behaving like this.

**Comment by ttshivers** [comment]
I figured it out. Looks like the API should be used like this instead: https://commitlint.js.org/#/reference-api

**Comment by ahmadnassri** [comment]
good catch! _(I mentioned this action wasn't fully readly, but I think with your help it's getting closer!)_

**Comment by ahmadnassri** [comment]
release is currently gated at 100% coverage limit in `npm ci` ...

**Comment by ttshivers** [comment]
I think it's missing one of these branches. https://github.com/ahmadnassri/action-conventional-commit-lint/blob/master/action/lib/lint.js#L11

Since config is static, I should probably remove the ternary. I just grabbed some code from the API example for that.

**Comment by ahmadnassri** [comment]
I'm looking at it now ...
